src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPresenter.java

```diff
@@ -64,6 +64,7 @@
 import org.rstudio.studio.client.workbench.views.output.find.model.FindInFilesServerOperations;
 import org.rstudio.studio.client.workbench.views.output.find.model.FindInFilesState;
 import org.rstudio.studio.client.workbench.views.output.find.model.FindResult;
+import org.rstudio.studio.client.workbench.views.plots.Plots;
```

Comment:
```suggestion
```

Reply:
I thought I already removed this stray import but seems like I may have missed it